File: java/Problem1036.java
import java.util.Scanner;

public class Problem1036 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double c = sc.nextDouble();

        double t = (b * b) - (4 * a * c);
        if (t < 0 || a == 0 ) {
            System.out.println("Impossivel calcular");
            sc.close();
            return;
        }
        double r1 = ((b * -1) + Math.sqrt(t))/(2 * a);
        double r2 = ((b * -1) - Math.sqrt(t))/(2 * a);
        
        System.out.printf("R1 = %.5f\n", r1);
        System.out.printf("R2 = %.5f\n", r2);

        sc.close();
    }
}

File: python/problem1036.py
valores = input().split(' ');

a = float(valores[0]);
b = float(valores[1]);
c = float(valores[2]);

# a, b, c = input().split

# ax² + bx + cx = 0
# @ =  b² - 4 . a . c
# (-b +- sqrt(@))/2 a

try:
	
	delta = (b ** 2) - (4 * a * c);
	
	bhaskara1 = ((b * -1) + (delta ** 0.5))/(2 * a);
	
	bhaskara2 = ((b * -1) - (delta ** 0.5))/(2 * a);
	
	print("R1 = %.5f" % bhaskara1);

	print("R2 = %.5f" % bhaskara2);

except:
	print("Impossivel calcular");

